How-to-get-the-MEAN-stack-running-locally-on-OSX.md

```diff
@@ -39,15 +39,15 @@ npm install -g express yo grunt grunt-cli generator-angular-fullstack bower
 ```
 
 ## Step 4: generating a fullstack app
-Make a directory for your Basejump projects. Assuming your desktop is your de facto workspace:
+Make a directory for your Back End Project projects. Assuming your desktop is your de facto workspace:
 ```sh
-mkdir ~/Desktop/basejumps && cd ~/Desktop/basejumps
+mkdir ~/Desktop/Back End Projects && cd ~/Desktop/Back End Projects
 ```
 Now that all the preparations are in place, it's time for the main event:
 ```sh
 yo angular-fullstack
 ```
-Answer the questions according to checklist items #13-23 of [Waypoint: Get Set for Basejumps](http://www.freecodecamp.com/challenges/waypoint-get-set-for-basejumps). Consult #24-27 if you run into errors. This will download ~350MB worth of files into your current directory.
+Answer the questions according to checklist items #13-23 of [Challenge: Get Set for Back End Projects](http://www.freecodecamp.com/challenges/Challenge-get-set-for-Back End Projects). Consult #24-27 if you run into errors. This will download ~350MB worth of files into your current directory.

```

Comment:
http://www.freecodecamp.com/challenges/get-set-for-our-back-end-development-projects